{"data_mtime":1754009448,"dep_lines":[13,14,12,15,49,11,5,6,8,10,1,1,1,1,1,1,1,1,1,1,1,1,18],"dep_prios":[5,5,5,5,20,5,10,5,5,10,5,30,30,30,30,30,30,30,30,30,30,30,25],"dependencies":["litellm.proxy.auth.auth_utils","litellm.proxy.db.exception_handler","litellm.proxy._types","litellm.types.services","litellm.proxy.proxy_server","litellm._logging","asyncio","typing","fastapi","litellm","builtins","_frozen_importlib","abc","litellm.types","litellm.types.llms","litellm.types.llms.base","pydantic","pydantic._internal","pydantic._internal._model_construction","pydantic.main","starlette","starlette.requests"],"hash":"8377612809b1d88a66bd8357f98d83b0b1e38536","id":"litellm.proxy.auth.auth_exception_handler","ignore_all":true,"interface_hash":"6c2dbc54cf84f1d431f658ff0b5dca9207855f89","mtime":1754008225,"options":{"allow_redefinition":false,"allow_redefinition_new":false,"allow_untyped_globals":false,"always_false":[],"always_true":[],"bazel":false,"check_untyped_defs":false,"disable_bytearray_promotion":false,"disable_error_code":[],"disable_memoryview_promotion":false,"disabled_error_codes":[],"disallow_any_decorated":false,"disallow_any_explicit":false,"disallow_any_expr":false,"disallow_any_generics":false,"disallow_any_unimported":false,"disallow_incomplete_defs":false,"disallow_subclassing_any":false,"disallow_untyped_calls":false,"disallow_untyped_decorators":false,"disallow_untyped_defs":false,"enable_error_code":[],"enabled_error_codes":[],"extra_checks":false,"follow_imports":"normal","follow_imports_for_stubs":false,"follow_untyped_imports":false,"ignore_errors":false,"ignore_missing_imports":true,"implicit_optional":false,"implicit_reexport":true,"local_partial_types":false,"mypyc":false,"old_type_inference":false,"platform":"darwin","plugins":[],"strict_bytes":false,"strict_concatenate":false,"strict_equality":false,"strict_optional":true,"warn_no_return":true,"warn_return_any":false,"warn_unreachable":false,"warn_unused_ignores":false},"path":"/opt/homebrew/lib/python3.9/site-packages/litellm/proxy/auth/auth_exception_handler.py","plugin_data":null,"size":4392,"suppressed":["opentelemetry.trace"],"version_id":"1.17.1"}